src/doc/ESMF_quickstart.tex

```diff
@@ -174,19 +174,19 @@ \subsubsection{Environment variables}
 
 \subsubsection{GNU make}
 The ESMF build system uses the GNU make program; it is normally named 
-{\tt gmake} but may also be simply {\tt make} or {\tt gnumake} on some 
-platforms (we will use {\tt gmake} in this document). ESMF does not use
+{\tt make} but may also be {\tt gmake} or {\tt gnumake} on some 
+platforms. We will use {\tt make} in this document. ESMF does not use
```

Comment:
I see similar wording in user_make.tex, which gets included in the "Building the ESMF Library" section of the "Building and Installing the ESMF" chapter. I'm thinking that should be changed, too, for consistency (e.g., adding, "We will use {\tt make} in this document." In fact, would it make sense to remove the first few sentences here and replace them with an inclusion of user_make.tex so that this piece stays consistent between the installation instructions and the quick start?

Reply:
I just reworked this, using `user_make.tex` in both places that discuss the GNU Make dependency. Thanks for pointing this out.